publications/recommender.tex

```diff
@@ -29,17 +29,20 @@
 
 \title{Microsoft/Recommenders: Best practices for building industry-grade recommender system}
 
-\author{\name Marina Meil \email mmp@stat.washington.edu \\
-       \addr Department of Statistics\\
-       University of Washington\\
-       Seattle, WA 98195-4322, USA
+\author{\name Le Zhang \email your@email.com \\
+       \addr One Microsoft Way \\
+       Redmond, WA 98052, USA
        \AND
-       \name Michael I.\ Jordan \email jordan@cs.berkeley.edu \\
-       \addr Division of Computer Science and Department of Statistics\\
-       University of California\\
-       Berkeley, CA 94720-1776, USA}
+       \name Jun-Ki Min \email jun.min@microsoft.com \\
+       \addr 1 Memorial Dr \\
+       Cambridge, MA 02142, USA
+       \AND
+       \name Authors \email your email \\
+       \addr your address line 1 \\
+       \addr your address line 2 \\
+       \addr your address line 3}
 
-\editor{Kevin Murphy and Bernhard Scholkopf}
+\editor{editors here}
```

Comment:
I don't think we need editors, do we?

Reply:
thought the publisher will assign editors when they publish. If not, yes we don't need.